src/jsonldparser.js

```diff
@@ -97,3 +96,17 @@ export default function jsonldParser (str, kb, base, callback) {
     .then(callback)
     .catch(callback)
 }
+
+/**
+ * statement quad depending on @type being a type node or a type value
+ * @param kb
+ * @param subject id
+ * @param property
+ * @param value
+ * @return quad statement
+ */
+function statementFromType(kb, id, property, value) {
```

Comment:
**praise**: Good catch! Thanks for fixing.

**thought**: However I find this code confusing, actually only understood what is the matter after looking at the test. Why is it called "FromType"? It is used for all properties, just handles `@type` differently. Perhaps just `createStatement` or `statementFromProperty`? Also, instead of heaving 2 ternaries it might be better readable to just differ between `if property === "@type"` and what happens `else`

Reply:
Thanks and agreed. Changes done

Reply:
Thank you very much!